src/components/CertificateTemplates/tlds/sg/edu/smu/degree-cert-template/certificate.js

```diff
@@ -0,0 +1,181 @@
+import PropTypes from "prop-types";
+import { get } from "lodash";
+
+const Template = ({ certificate }) => {
+  // Declaring what variables will be available to the template from the certificate
+  const certificateName = get(certificate, "name");
+  const honors = get(certificate, "additionalData.honors", "\u00a0");
+  const recipientName = get(certificate, "recipient.name");
+  const awardDate = get(certificate, "additionalData.awardDate");
+  const bkgrdImg = get(certificate, "additionalData.bkGrd");
```

Comment:
don't put static images in the opencert file as it makes it very big, put it in the template instead